src/ws_server.cu

```diff
@@ -416,6 +418,7 @@ class AprilTagHandler : public seasocks::WebSocket::Handler {
   std::atomic<bool> settings_changed_{false};
   std::atomic<bool> rotate_img_{false};
   std::thread read_thread_;
+  json empty_detections_record;
```

Comment:
no need to make this a private member - just declare it at the top of the while loop before line 327

Reply:
added